modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -1,12 +1,11 @@
 = Field Level Encryption from the SDK
-:description: The Field Level Encryption library enables encryption and decryption of JSON fields, to support FIPS-140-2 compliance.
+:description: The Field Level Encryption library enables encryption and decryption of JSON fields, to support FIPS-140-2 compliance.
 :page-topic-type: howto
 :page-edition: Enterprise Edition
-:page-status: Developer Preview
-// // :page-aliases: ROOT:encrypting-using-sdk.adoc
+:page-aliases: ROOT:encrypting-using-sdk.adoc

```

Comment:
another uncommented page-aliases - as per Richard's comment on other one, the practice is to uncomment if doing so doesn't break the build?

Reply:
See comment on the other change.